src/env-inl.h

```diff
@@ -736,7 +736,7 @@ inline void Environment::SetMethod(v8::Local<v8::Object> that,
   const v8::NewStringType type = v8::NewStringType::kInternalized;
   v8::Local<v8::String> name_string =
       v8::String::NewFromUtf8(isolate(), name, type).ToLocalChecked();
-  that->Set(name_string, function);
+  that->Set(context, name_string, function).FromJust();
```

Comment:
Shouldn't this return if it's empty? Also other `FromJust` below

Reply:
@joyeecheung I'm not sure that it matters in these two cases.

Reply:
I don’t think we can really signal an error here without making `SetMethod()` itself return a `Maybe<>`…

Reply:
@addaleax hmm, yeah, the caller can handle that as well, but this is mostly used in `Initialize` anyways